acnportal/acnsim/network/current.py

```diff
@@ -26,7 +26,9 @@ def __init__(self, loads=None):
         elif isinstance(loads, pd.Series):
             super().__init__(loads)
         elif loads is None:
-            super().__init__()
+            # The object type is specified explicitly here to address a
+            # warning in pandas 1.x.
+            super().__init__(dtype='float64')
```

Comment:
In Pandas 1.0.1, the default type for series is 'object', so for forward compatibility I've explicitly specified the data type here.

Reply:
Good catch!